bundle/manifests/atlas.mongodb.com_atlasbackuppolicies.yaml

```diff
@@ -97,6 +98,7 @@ spec:
                       - days
                       - weeks
                       - months
+                      - years
```

Comment:
@igor-karpukhin @helderjs @josvazg @roothorp this formally is not a bug fix but an added feature so I am thinking if we should declare this a 2.2.